components/navigation/ui/src/main/resources/PhenoTips/DatabaseFilters.xml

```diff
@@ -61,6 +61,7 @@
 #if ($activeFields.contains('external_id'))
 * By **external identifier**: {{html clean=false}}&lt;p class="xHint"&gt;(The hospital ID manually entered)&lt;/p&gt;&lt;input type="text" name="external_id" value=""/&gt;{{/html}}
 #end
+* Which privacy levels would you like to include in your export? {{html clean=false}}&lt;br&gt; #foreach ($visibility in $services.permissions.listVisibilityOptions()) &lt;input type="checkbox" class="multipleSelection privacyExport" name="visibility" value="$visibility" #if ($visibility != 'hidden')checked#end/&gt;$visibility&lt;br&gt;#end{{/html}}
```

Comment:
Going with the style of the rest of the form, the message should probably be "By **privacy level**".